### ISSUE TITLE
Support multiple links for a single entry

### ISSUE THREAD

**Issue by ohadschn** [4tal/LinksForIsrael#116]
For example, see: https://github.com/4tal/LinksForIsrael/issues/115
There is a hotline number, as well as an e-mail
Either one can be linked, but not both
We can either support it natively (similar to the multiple share links we have for categories), or link to some "bio" site with contact information where there are links to e-mail, whatsapp, phone# etc

**Comment by ohadschn** [comment]
@4tal I would say the current implementation is partial, as it allows only one link per type.
I think a better implementation would be something like
```json
{
    "name": "foo",
    "displayName": "FOO help",
    "shortDescription": "Helping foo",
    "links": [
        {
            "type": "facebook",
            "url": "https://m.facebook.com/groups/780925150471197/?ref=share&mibextid=lURqYx"
        },
        {
            "type": "whatsapp",
            "hover": "support whatsapp",
            "url:": "https://chat.whatsapp.com/Bsy4Sc9jM4PIS64JD1aRnp"
        },
        {
            "type": "whatsapp",
            "hover": "recruiting whatsapp",
            "url:": "https://chat.whatsapp.com/xkjhl1kjh4PIS64JD1aRnp"
        }
    ]
}
```

**Comment by 4tal** [comment]
Current model can support and it's kind of working like that. Can you suggest situation where we'll need to use id per link in the near future? or use case current format don't support.

**Comment by ohadschn** [comment]
> Current model can support and it's kind of working like that. Can you suggest situation where we'll need to use id per link in the near future? or use case current format don't support.

I don't follow, I did exactly that in the comment above - use case that isn't supported and is needed...

**Comment by chiptus** [comment]
I think this is related to https://github.com/4tals/LinksForIsrael/issues/206 

at least to the bottom line:
> or example, anything social will automatically get [instagram, twitter, facebook, tiktok])

**Comment by ohadschn** [comment]
> I think this is related to #206
> 
> at least to the bottom line:
> 
> > or example, anything social will automatically get [instagram, twitter, facebook, tiktok])

You mean moving the tag to the link (rather than the initiative) level? Something like
```json
"links": [
        {
            "type": "facebook",
            "url": "https://m.facebook.com/groups/780925150471197/?ref=share&mibextid=lURqYx",
            "tags": ['facebook']
        },
```
Are you sure that makes sense? To me it sounds like tags should remain at the initiative level - for example, when used for search what you'll find is the initiative, not any specific link...

**Comment by ohadschn** [comment]
@4tal I think this became more important when agriculture groups started to be involved. Many of those initiatives are basically a collection of WhatsApp groups, so we need multiple WhatsApp links for the same initiative.

**Comment by ohadschn** [comment]
@omer-friedman if you're thinking about the next issue to tackle, may I suggest this?

Looking at the rides / transportation category, again there are basically a million groups, and we could just have one "initiative" called "whatsapp groups" with multiple whatsapp links, where something like a hover over each one gives you its name and short description

The same could be done for general assistance, strategic communication, agriculture (and as mentioned above)...

**Comment by omer-friedman** [comment]
It would be great if we'll have a UX/UI for this one.
@4tal can you talk with vitali about it?

**Comment by ohadschn** [comment]
> It would be great if we'll have a UX/UI for this one. @4tal can you talk with vitali about it?

I don't think we have to wait, IMO even something as simple as having the same icon multiple time with a differentiating hover would be better than no support at all...
